Sources/PointFree/Blog/BlogAtomFeed.swift

```diff
@@ -28,7 +28,7 @@ private let feedView = View<[BlogPost]> { posts in
 private func atomEntry(for post: BlogPost) -> AtomEntry {
   return AtomEntry(
     content: blogPostContentView.view(post),
-    siteUrl: url(to: .blog(.show(post))),
+    siteUrl: "", // TODO: url(to: .blog(.show(post))),
```

Comment:
I was just commenting out blog related stuff to get errors down, but this is an easy one to fix.

Reply:
and this